CHANGES.md

```diff
@@ -1,5 +1,32 @@
 # Changes
 
+## edge-21.11.4
+
+This edge release introduces a change in the destination service to honor
+opaque ports set in the `proxyProtocol` field of `Server` resources. This
+change makes it possible to set opaque ports directly in `Server` resources
+without needing the opaque ports annotation on pods. Version `v1alpha1` of the
```

Comment:
TIOLI: `v1alpha1` was just a edge thing right? This sentence makes it sound a bit more important than reality. 🤔

Reply:
Hm, yeah, can take it out. It's listed as a bullet point anyway, doesn't have to be in the overview.